### PR TITLE
chore/Add gh workflow

### PR DESCRIPTION
**Pull request by Oolay**
- adds github action to run CI (just jest tests)

I had to remove and re-add the game folder to rename it as Github did not recognise the change and the CI was failing. This is from when we renamed the folder a while back.

### CODE REVIEW

**Comment by Thimey** [comment]
> I had to remove and re-add the game folder to rename it as Github did not recognise the change and the CI was failing. This is from when we renamed the folder a while back.

You should be able to do this with `git mv src/game src/Game`

**Review by Thimey — APPROVED** [review]
Nice work - lgtm!
Just wondering if we should try to run actions as little as possible for now since we are only on free tier

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -0,0 +1,28 @@
+name: Continuous Integration
+
+on:
+  push:
+    branches:
+      - '**'
+      - '!main'
```
[the remark this answers is not in the document]
Comment (Oolay) [reply .github/workflows/ci.yml:7]: hmm I can't see how to do it on push only for open PRs 🤔 . I'll just set it to run when a PR is opened - should be OK for now

[the remark this answers is not in the document]
Comment (Oolay) [reply .github/workflows/ci.yml]: actually, maybe that's done it 👍

**Pull request merged**